source/fundamentals/crud/read-operations/text.txt

```diff
@@ -0,0 +1,237 @@
+
+.. literalinclude:: /includes/fundamentals/code-snippets/SearchText.java
+   :language: java
+   :dedent:
+   :start-after: begin textIndex
+   :end-before: end textIndex
+
+If there was another string field, ``description``, you wanted to index,
+you can create a compound index to perform a text search in both fields
+using the following snippet: 
+
+.. code-block:: java
+
+    collection.createIndex(Indexes.compoundIndex(Indexes.text("course"), Indexes.text("description")));
+
+For more information, see our server manual entry on :manual:`text
+indexes </core/index-text/>`.
+
+Searching
+---------
+
+To perform a text search, use the ``text()`` method of the ``Filters``
+builder. 
+
+The ``text()`` method performs a text search on the fields indexed with
+a text index. The text search matches documents which contain the
+specified search term. 
+
+You can include ``TextSearchOptions`` as the second parameter to specify
+additional text search options to handle case sensitivity, diacritic
+sensitivity and the language of the text. 
+
+For more information, see the API Documentation on the
+:java-core-api:`text()
+<com/mongodb/client/model/Filters.html#text(java.lang.String)>` method
+and :java-core-api:`TextSearchOptions<com/mongodb/client/model/TextSearchOptions.html>`.
+
+By a Term
+~~~~~~~~~
+
+To search for a term in your text index, pass the term as a string
+to the ``text()`` method. 
+
+If you want to match multiple terms in your text index, separate your
+terms with spaces. The ``text()`` method performs a logical OR on
+each term and returns documents that contain any of the terms. 
+
+Example 
+```````
+
+The student would like to see which courses mention "Computer" in their
+name. 
+
+To address this scenario, the student performs a text search on
+documents that contain the term "Computer": 
+
+.. literalinclude:: /includes/fundamentals/code-snippets/SearchText.java
+   :language: java
+   :dedent:
+   :start-after: begin wordExample
+   :end-before: end wordExample
+
+The following shows the output of the code above:
+
+.. code-block:: json
+   :copyable: false
+
+   { "_id": 2, "course": "Discrete Mathematical Structures for Computer Science" }
+   { "_id": 4, "course": "Computer Science" }
+   { "_id": 5, "course": "Computer Hacking Revealed" }
+   { "_id": 6, "course": "Advanced Computer Game Programming" }
+   { "_id": 7, "course": "Image Processing in Computer Science" }
+
+.. note::
+
+    The output of the snippet above is equivalent to performing the
+    following text search:
+
+    .. literalinclude:: /includes/fundamentals/code-snippets/SearchText.java
+       :language: java
+       :dedent:
+       :start-after: begin multiWordExample
+       :end-before: end multiWordExample
+
+By a Phrase
+~~~~~~~~~~~
+
+To search for a phrase in your text index, pass the phrase with escaped quotes
+to the ``text()`` method.  
+
+If you don't add escaped quotes, the method performs a logical OR on the
+terms listed.
```

Comment:
It might make more sense to the reader to mention that it performs a term search without the escaped quotes. It's a little easier to read than an explanation of what the term search does.